### ISSUE TITLE
[Enhancement] Call BeefSetup.exe from command line (No GUI)

### ISSUE THREAD

**Issue by RogueMacro** [beefytech/Beef#438]

**Comment by bfiete** [comment]
What is the use case for that?

**Comment by RogueMacro** [comment]
Automate installation

**Comment by bfiete** [comment]
Why do you want to automate installation?

**Comment by RogueMacro** [comment]
So I don't have to go to a url, wait for the download, then run it. Easier to just run a script that does it for me. Was also thinking of maybe adding it as a future to the package manager. idk. Might. Anyway, it's in my nature to do so.

**Comment by bfiete** [comment]
Getting rid of the UI isn't important then - just removing the need to click on the actual 'Install' and 'Done' buttons would be sufficient.

**Comment by RogueMacro** [comment]
Yeah. As long as it automatically where my previous Beef installation is.
